listenbrainz/domain/spotify.py

```diff
@@ -0,0 +1,102 @@
+import pytz
+from flask import current_app, url_for
+import spotipy.oauth2
+
+from listenbrainz.db import spotify as db_spotify
+import datetime
+
+
+class Spotify:
+    # TODO: include musicbrainz id (username)
+    def __init__(self, user_id, user_token, token_expires, refresh_token,
+                 last_updated, active, update_error_message):
+        self.user_id = user_id
+        self.user_token = user_token
+        self.token_expires = token_expires
+        self.refresh_token = refresh_token
+        self.last_updated = last_updated
+        self.active = active
+        self.update_error_message = update_error_message
+
+    def get_spotipy_client(self):
+        return spotipy.Spotify(auth=self.user_token)
+
+    def nice_last_updated(self):
```

Comment:
Is this a function to make it nice and human readable? If so, maybe should also also use an "ago" function to make the timestamp more approachable.

Reply:
I think this was for a potential status message on the user profile page ("You have connected your Spotify account to ListenBrainz and it was last updated x", 'never' is for the case where a user connects but it hasn't run yet) - you're right that "ago" would be a good update here.